compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -2578,6 +2578,8 @@ object Types {
     private var myRef: Type = null
     private var computed = false
 
+    def pending(using Context): Boolean = computed && myRef == null
```

Comment:
This looks identical to `def evaluating` below ?

Reply:
Indeed.